### PR TITLE
set max version to translate study

### PR DESCRIPTION
**Pull request by atuchin-m**
Resolves https://github.com/brave/brave-variations/issues/403
PR to main: https://github.com/brave/brave-variations/pull/405

### CODE REVIEW

File: seed/seed.json
Comment (atuchin-m) [inline seed/seed.json:1227]: Here is the proof that v1.45.44 has the commit to enable the feature:
https://github.com/brave/brave-core/compare/v1.45.43...v1.45.44

I've set 106, not 105 because the final 1.45.x builds use cr106.

**Review by iefremov — APPROVED** [review]

**Review by goodov — APPROVED** [review]

**Review by kjozwiak — APPROVED** [review]
Uplift into `production` approved after deliberating with @brave/uplift-approvers. QA has verified the PR as per https://github.com/brave/brave-variations/pull/405#issuecomment-1294230813 & https://github.com/brave/brave-variations/pull/405#issuecomment-1294354339.

**Pull request merged**
